.github/actions/usd-install-dep/action.yml

```diff
@@ -31,7 +31,9 @@ runs:
       if: steps.cache-usd.outputs.cache-hit != 'true'
       working-directory: ${{github.workspace}}/dependencies/usd
       shell: bash
-      run: patch -p1 < $GITHUB_ACTION_PATH/0001-usd-onetbb.patch
+      run: |
+        patch -p1 < $GITHUB_ACTION_PATH/0001-usd-onetbb.patch
+        patch -p1 < $GITHUB_ACTION_PATH/0002-install-bin.patch
```

Comment:
maybe open an issue upstream ?

Reply:
Forgot about it, but I've already opened one: https://github.com/PixarAnimationStudios/OpenUSD/issues/2490